README.md

```diff
@@ -228,6 +229,13 @@ version.terminator
 version.whodunnit
 version.version_author
 
+# Returns user active user model instance
+version.user
+# This method takes two arguments. First is the model name and the second is
+# primary key. By default the model is `User` and the primary key is `id`.
+# If you are using a different model (example: Customer) to store users then
+# make a call like this: version.user(:Customer)
```

Comment:
Please move these detailed comments above the method definition, in `version_concern.rb`.